src/components/PGN.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <div :class="theme">
+    <div class="btn-flex">
+      <button @click="flipBoard">Flip Board</button>
+    </div>
```

Comment:
Could you remove the div wrapper and move the flip button to be the first element in the btn-flex with the < << > >> arrow controls? (About line 82ish).